src/classes/CaseTriggerHandlerTest.cls

```diff
@@ -0,0 +1,108 @@
+/**
+ * Created by Synebo on 21.09.2021.
+ */
+@IsTest
+public with sharing class CaseTriggerHandlerTest {
+
+    @IsTest static void caseTriggerTests() {
```

Comment:
test method should have the name of the method + 'Test'. In this test it should be countCaseUpdateTest